components/configuring_network_bridge.mdx

```diff
@@ -0,0 +1,46 @@
+Your is configured to bridge its physical ethernet ports together. This allows any ethernet port to be used
+as a connection to the internal `192.168.131.1/24` network for connecting sensors, diagnostic equipment, or
+manipulators, or for connecting the your robot to the internet for the purposes of installing updates.
+
+In the unlikely event you must modify your robot's ethernet bridge, you can do so by editing the configuration
+file found at `/etc/netplan/50-clearpath-bridge.yaml`:
+
+```bash
+# Configure the wired ports to form a single bridge
+# We assume wired ports are en* or eth*
+# This host will have address 192.168.131.1
+network:
+version: 2
+renderer: networkd
+ethernets:
+bridge_eth:
+  dhcp4: no
+  dhcp6: no
+  match:
+    name: eth*
+bridge_en:
+  dhcp4: no
+  dhcp6: no
+  match:
+    name: en*
+bridges:
+br0:
+  dhcp4: yes
+  dhcp6: no
+  interfaces: [bridge_en, bridge_eth]
+  addresses:
+    - 192.168.131.1/24
+```
+
+This file will create a bridged interface called br0 that will have a static address of `192.168.131.1`,
```

Comment:
Consider adding emphasis to 'br0' with italics

Reply:
or with `` similar to `en*` below

Reply:
Done